### PR TITLE
Gitbook deployment guide

### PR DESCRIPTION
**Pull request by terrikwak**
## Proposed changes

Deployment Guide (Just the Klaytn IDE section)
GitBook [link](https://app.gitbook.com/o/-LZ3M2TduHoL5IluFRpr/s/YtyyxJ07x3JDGpj6dJ5S/smart-contract/deploy-guide)

## Types of changes

Please put an x in the boxes related to your change.

- [x] Minor Issues and Typos
- [ ] Major Content Contribution
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to reach out. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-docs/blob/master/CONTRIBUTING.md)
- [x] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn-docs)
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large content contribution, kick off the discussion by explaining why you would suggest the content contribution, etc...

### CODE REVIEW

**Comment by terrikwak** [comment]
@kjhman21  I added the explanation for the contract button colors :) Please take a look when you have time

**Review by kjhman21 — APPROVED** [review]
LGTM. Thanks for the update.

File: docs/smart-contract/deploy-guide.md [hunk 1]
```diff
@@ -39,31 +39,29 @@ contract KlaytnGreeter is Mortal {
     }
 }
 ```
-- Set compiler version. Currently, you can choose between 0.4.24 and 0.5.6. (Default is 0.5.6.)  
-
-![](img/deploy-with-ide/deploy-with-ide.002.png)
 
+- Select Compiler in the icon panel. Choose the desired EVM environment. For the Klaytn networks, you can choose between Baobab (testnet) and Cypress (mainnet). And click Compile.
```
Comment (neoofklaytn) [inline docs/smart-contract/deploy-guide.md:43]:
```suggestion
- Select Compiler in the icon panel. Choose the desired EVM environment. For the Klaytn networks, you can choose between Baobab (testnet) and Cypress (mainnet). Click `Compile` when the sample code is ready to be complied before actual deployment.
```

File: docs/smart-contract/deploy-guide.md [hunk 2]
```diff
@@ -39,31 +39,29 @@ contract KlaytnGreeter is Mortal {
...
 
-- Click Run tab. At `Environment` dropdown, you can choose a target network to deploy the contract. (Baobab is test network and Cypress is main network.)
+![](img/deploy-with-ide/02_deployment_compile.png)
 
-![](img/deploy-with-ide/deploy-with-ide.003.png)
+- Now we can deploy the contract. Click on the Klaytn logo in the icon panel. Import an account by clicking the plus button next to `Account`. Make sure that the account has sufficient KLAY to deploy and transact the contracts.
```
Comment (neoofklaytn) [inline docs/smart-contract/deploy-guide.md:47]:
```suggestion
- Now we can deploy the contract. Click on the Klaytn logo in the icon panel. Import an account by clicking the plus button next to `Account`. Make sure that the account has sufficient KLAY to pay for the transaction of deploying the smart contracts required.
```

File: docs/smart-contract/deploy-guide.md [hunk 3]
```diff
@@ -39,31 +39,29 @@ contract KlaytnGreeter is Mortal {
...
 - Set Gas limit and Value to send. 
   - You may need to set higher Gas limit if you are deploying a more complicated contract. In this example, you can leave it as it is.
   - Set `Value` to 0 unless you want to send `KLAY` to the contract at the time of deployment.
-- Enter "Hello World!" as an argument for constructor function and click on `Deploy` button. If everything goes successful, deployed contract's instance will show up below.
+- Enter "Hello World!" as an argument for constructor function and click on `Deploy` button.
+
+![](img/deploy-with-ide/03_deployment_hello.png)
 
-![](img/deploy-with-ide/deploy-with-ide.005.png)
+- If the contract is successfully deployed, you will see the result in the terminal.
```
Comment (neoofklaytn) [inline docs/smart-contract/deploy-guide.md:58]:
```suggestion
- If the contract is successfully deployed, you will see the corresponding transaction receipt and detailed result in the terminal. 
```

File: docs/smart-contract/deploy-guide.md [hunk 4]
```diff
@@ -39,31 +39,29 @@ contract KlaytnGreeter is Mortal {
...
+- Enter "Hello World!" as an argument for constructor function and click on `Deploy` button.
+
+![](img/deploy-with-ide/03_deployment_hello.png)
 
-![](img/deploy-with-ide/deploy-with-ide.005.png)
+- If the contract is successfully deployed, you will see the result in the terminal. 
 
-- You can interact with the contract by clicking on the function buttons. Dark blue buttons are `send` functions which can change state on blockchain and consumes gas. Light blue buttons are `call` functions which do not change state and do not require gas fee.   
+- You can interact with the contract by clicking on the function buttons. The functions are represented in different colors. `constant` or `pure` functions in Solidity have blue bottons (`greet` in the example) and do not create a new transaction, so they don't cost any gas. Red buttons (`kill` in the example) represent `payable` functions that change the state on the blockchain, consumes gas and can accept value. Orange buttons are for `non-payable` functions that change the contract state but do NOT accept a value.
```
Comment (neoofklaytn) [inline docs/smart-contract/deploy-guide.md:60]:
```suggestion
- You can interact with the contract by clicking on the function buttons. The functions are represented in different colors. `constant` or `pure` functions in Solidity have blue bottons (`greet` in the example) and do not create a new transaction, so they don't cost any gas. Red buttons (`kill` in the example) represent `payable` functions that change the state on the blockchain, consume gas and can accept value. Orange buttons are for `non-payable` functions that change the contract state but do NOT accept a value.
```

**Review by dcground — APPROVED** [review]

**Pull request merged**